cmd/minishift/cmd/addon/addons_list.go

```diff
@@ -63,13 +64,13 @@ func init() {
 	var err error
 	defaultListTemplate, err = template.New("list").Parse(defaultAddonListFormat)
 	if err != nil {
-		glog.Errorln("Error creating list template:", err)
+		fmt.Println("Error creating list template: ", err)
```

Comment:
Ok, let's look at this once more. This is over and over the same pattern. Log somethings and then call _Exit_. What's about making the code better? If we either

* make atExit a variadic function, aka _Exit(exitCode int, messages ...string)
* our by adding _atexit.ExitWithMessage(exitCode int, message string)

We encapsulate the actual logging and how we do it in one single spot. If the then later, for example as part of #169 want to use a logger, we only have to change this at one single spot, instead of touching 20 files again.

The second option is most probably the better, since in the first we would need to decide what to do with multiple message and what it actually means.